sympy/simplify/simplify.py

```diff
@@ -994,7 +994,7 @@ def goodlog(l):
                 num, den = k, -k
                 if num.count_ops() > den.count_ops():
                     num, den = den, num
-                other.append(num*log(log1.pop(num).args[0]/log1.pop(den).args[0]))
+                other.append(num*log(log1.pop(num).args[0]/log1.pop(den).args[0] , evaluate = False))
```

Comment:
This line is too long, it should be split.

Reply:
@jksuom should I do it by adding 2 variables?

Reply:
I think it could be done as shown in a [comment](https://github.com/sympy/sympy/pull/14070#issuecomment-362900475) above. (There would be no need to think of new names for the two variables.)